go.mod

```diff
@@ -0,0 +1,5 @@
+module go-rest-api
```

Comment:
Just as a nitpick, even if this package name will work fine, the convention to name your project is usually how you will see from external packages: `github.com/<ownerOfTheProject>/<projectName>`